extensions/machine-learning/src/views/models/manageModels/currentModelsComponent.ts

```diff
@@ -58,53 +63,84 @@ export class CurrentModelsComponent extends ModelViewBase implements IPageView {
 		this._loader = modelBuilder.loadingComponent()
 			.withItem(formModelBuilder.component())
 			.withProperties({
-				loading: true
+				loading: true,
 			}).component();
+		this._dataStateImageComponent = modelBuilder.image().withProperties({
+			iconPath: this.asAbsolutePath('images/emptyState.svg'),
+			iconHeight: '128px',
+			iconWidth: '128px',
+			height: '128px'
+		}).component();
 		this._labelComponent = modelBuilder.text().withProperties({
-			width: 200,
 			value: constants.modelsListEmptyMessage
 		}).component();
 		this._descriptionComponent = modelBuilder.text().withProperties({
-			width: 200,
 			value: constants.modelsListEmptyDescription
 		}).component();
 		this._labelContainer = modelBuilder.flexContainer().withLayout({
 			flexFlow: 'column',
-			width: 800,
+			width: '750px',
 			height: '400px',
-			justifyContent: 'center'
+			justifyContent: 'flex-start',
+			textAlign: 'center'
+		}).component();
+		this._subheadingContainer = modelBuilder.flexContainer().withLayout({
+			flexFlow: 'column',
+			width: '452px'
+		}).component();
+		this._subheadingTextComponent = modelBuilder.text().withProperties(<azdata.CheckBoxProperties>{
+			value: constants.viewImportModelsDesc,
+			CSSStyles: {
+				'font-size': '13px'
+			}
+		}).component();
+		this._subheadingLinkComponent = modelBuilder.hyperlink().withProperties({
+			label: constants.learnMoreLink,
+			url: 'https://www.microsoft.com/',
```

Comment:
the link is not correct. should we atleast link to main doc of ml until the doc is ready?

Reply:
Placeholder until I know differently.